src/components/Popover/README.md

```diff
@@ -412,3 +412,38 @@ Use when you have few actions that affects the whole page. Action sheets doesn
 
 - To put a list of actions in a popover, [use the action list component](/components/actions/action-list)
 - To let merchants select simple options from a list, [use the select component](/components/forms/select)
+
+---
+
+## Accessibility
+
+<!-- content-for: android -->
+
+See Material Design and development documentation about accessibility for Android:
+
+- [Accessible design on Android](https://material.io/design/usability/accessibility.html)
+- [Accessible development on Android](https://developer.android.com/guide/topics/ui/accessibility/)
+
+<!-- /content-for -->
+
+<!-- content-for: ios -->
+
+See Apple’s Human Interface Guidelines and API documentation about accessibility for iOS:
+
+- [Accessible design on iOS](https://developer.apple.com/design/human-interface-guidelines/ios/app-architecture/accessibility/)
+- [Accessible development on iOS](https://developer.apple.com/accessibility/ios/)
+
+<!-- /content-for -->
+
+<!-- content-for: web -->
+
+Popovers usually contain an [option list](/components/lists-and-tables/option-list) or an [action list](/components/actions/action-list), but can also contain other controls or content.
+
+### Keyboard support
+
+- When a popover opens, focus moves to the popover container
```

Comment:
Should we mention the `preventAutoFocus` prop at all? I'm thinking there's opportunity here to make it clear when it's okay to use, like when autoFocus would prevent merchants from completing the action that triggers the popover. For example, when a [text field listening for certain characters is the activator](https://github.com/Shopify/web/pull/12272) and autofocusing focusable items inside the activated popover would remove focus from the text field while typing; additional focus management needs to be implemented in this case.

<details>

<summary>(Click to view gif of example case described above)</summary>


![2019-05-07 16 32 42](https://user-images.githubusercontent.com/18447883/57331178-fc0c5880-70e5-11e9-9a0f-04b35d2fbce2.gif)

</details>

Reply:
This came up in discussion about popovers today too, actually. Based on that and other questions about keyboard accessibility, I think I want to chunk out a separate page (probably under patterns and guides?) about keyboard interactions in general. Would love your thoughts on doing it that way, @chloe!

Reply:
I think that's a great idea, as it comes up in a lot of our components 👍